metasrc/tex/lsstdoc.py

```diff
@@ -2,7 +2,15 @@
 
 __all__ = ['LsstDoc']
 
+import logging
+import os
+
 from .commandparser import LatexCommand
+from ..pandoc.convert import convert_lsstdoc_tex
+from .scraper import get_macros
+from .normalizer import read_tex_file, replace_macros
+from .lsstbib import get_bibliography, KNOWN_LSSTTEXMF_BIB_NAMES
+from .citelink import CitationLinker
 
 
 class LsstDoc(object):
```

Comment:
Minor aside: If this class is solely for latex documents maybe the class should be `LsstLatexDoc()`? This is obvious on the latex side where we call it `lsstdoc.cls` but on the Python side it looks like a generic LSST Document.